test/commands/build.test.ts

```diff
@@ -39,3 +40,33 @@ describe("Init and build", function () {
     expect(buildResults[0].releaseMultiHash).to.include("/ipfs/Qm");
   });
 });
+
+describe("Init and build template repo", function () {
+  this.timeout(120 * 1000);
+
+  before("Clean testDir", () => cleanTestDir());
+  after("Clean testDir", () => cleanTestDir());
+
+  before("Init repo", async () => {
+    await initHandler({
+      dir: testDir,
+      force: true,
+      yes: true,
+      template: true
+    });
```

Comment:
I would expect some checks such as the directories created, files etc

Reply:
Done